server/src/__tests__/fixtures/semanticTokens.bb

```diff
@@ -0,0 +1,14 @@
+FOO = 'I am a fool'
```

Comment:
Maybe you can use a more neutral message here. "bar"

Reply:
Alright. neutral it is.